src/MockHttpClient/MockHttpClient.csproj

```diff
@@ -1,13 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.4;net462</TargetFrameworks>
```

Comment:
This will drop nuget support for net462 and net1.4

Although i'm not against the idea of dropping support for them, it should be done for a reason (e.g enable new features, bugs in old frameworks etc)